capybara-201611301443567584050390.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Delete these capybara-23402734 files. They are the saved screenshots from when you did `save_and_open_page`. They should not be part of your final code.